packages/app-project/src/screens/ProjectHomePage/components/ZooniverseTalk/components/RecentSubjects/RecentSubjectsCarousel.js

```diff
@@ -0,0 +1,39 @@
+import counterpart from 'counterpart'
+import { arrayOf, shape, string } from 'prop-types'
+import React from 'react'
+import { Carousel } from 'grommet'
+import SubjectThumbnail from './components/SubjectThumbnail'
+
+// TODO: Use the subject viewers from the classifier
+function RecentSubjectsCarousel (props) {
+  const { className, href, subjects } = props
+  const height = 500
+  const width = 700
+  return (
+    <Carousel
+      controls='arrows'
+      fill
+    >
+      {subjects.map( subject => (
+        <SubjectThumbnail
+          key={subject.id}
+          height={height}
+          href={href}
+          subject={subject}
+          width={width}
+        />
+      ))}
+    </Carousel>
+  )
+}
+
+RecentSubjectsCarousel.propTypes = {
+  subjects: arrayOf(shape({
+    id: string
+  }))
+}
+
+RecentSubjectsCarousel.defaultProps = {
+}
```

Comment:
Should we have default props here? Or set some to be required?

Reply:
`subjects` should be required, I think, and `className` should default to undefined. `href` I'm not sure about because the links don't actually work in the Grommet carousel.